packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx

```diff
@@ -74,6 +79,40 @@ export class GuxPopoverList {
   @Event()
   guxdismiss: EventEmitter<void>;
 
+  /**
+   * Fired when the popover is open
+   */
+  @Event()
+  guxopen: EventEmitter<void>;
+
+  /**
+   * Fired when the popover is close
+   */
+  @Event()
+  guxclose: EventEmitter<void>;
+
+  @Watch('isOpen')
+  onIsOpenChange(isOpen: boolean): void {
+    if (isOpen) {
+      this.guxopen.emit();
+
+      console.log('Focus', this.root);
+      afterNextRender(() => {
+        this.root.focus();
+      });
+    } else {
+      this.guxclose.emit();
+    }
+  }
+
+  @Listen('keydown')
+  onKeyDown(event: KeyboardEvent): void {
+    switch (event.key) {
+      case 'Escape':
+        this.dismiss();
```

Comment:
Focus is lost after pressing `esc`. Normally we would expect the focusable 'target' element to gain focus after the popover is closed. For an example, see the focus behavior in `gux-modal`

Reply:
Might have to go to UX with this as we currently have no guidance for how a popover should be triggered. In the example I went with on `mouseover` and on `focus`. That means that if I focus the target element on dismiss it will just open it again.